backend/api/Startup.cs

```diff
@@ -82,6 +83,12 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.Default.IgnoreNonMapped(true);
                 options.Default.IgnoreNullValues(true);
+                options.AllowImplicitDestinationInheritance = true;
+                options.AllowImplicitSourceInheritance = true;
+                options.Default.UseDestinationValue(member =>
```

Comment:
This allows us to access and update collections without a setter.  @devinleighsmith you won't need to have a `private set;` anymore.

Reply:
nice